unstructured_ingest/v2/processes/connectors/fsspec/fsspec.py

```diff
@@ -166,6 +166,7 @@ def run(self, **kwargs: Any) -> Generator[FileData, None, None]:
 
             additional_metadata = self.sterilize_info(file_data=file_data)
             additional_metadata["original_file_path"] = file_path
+            additional_metadata["record_display_name"] = file_path
```

Comment:
We should make this a concrete field on the FileData class, set to an optional string defaulted to None.

Reply:
@rbiseck3 why is this preferred?

Reply:
It’ll provided a common interface that can be taken advantage of downstream from this. There’s no contractual guarantee on the data that lives in the additional metadata blob.